### PR TITLE
feat: Add pre-commit and TFLint configuration files

### PR DESCRIPTION
**Pull request by migueldelucasdoit**
@mvaude @tibor-szalai please check this PR for pre-commit hooks.
@mvaude when you merge your PR  https://github.com/doitintl/platform-iac/pull/6 I will update README.md file with requirements.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+repos:
+- repo: https://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.76.0 # Get the latest from: https://github.com/antonbabenko/pre-commit-terraform/releases
+  hooks:
```
Comment (mvaude) [inline .pre-commit-config.yaml:4]: should we make files explicit for these hooks like for `cloudformation`?
e.g. for each hook:
```
files: terraform/.*\.(tf|hcl)$
```
Without scope defined they will run for each sub-directory, won't they?

**Review by mvaude — APPROVED** [review]
Question regarding pre-commits hooks scope. Otherwise LGTM :+1:

File: .pre-commit-config.yaml (hunk 1 above)
Comment (migueldelucasdoit) [reply .pre-commit-config.yaml:4]: I guess each hook already has a defined set of files that are whitelisted. You can check them here
https://github.com/antonbabenko/pre-commit-terraform/blob/master/.pre-commit-hooks.yaml
If you feel we should be more restrictive (and less flexible) we can also include the folder in all hooks configurations

Comment (mvaude) [reply .pre-commit-config.yaml:4]: You are right, let's make some bench testing when we have more files in the repository :+1:

**Pull request merged**